apps/vsce/src/components/engineService.ts

```diff
@@ -45,6 +45,9 @@ export const Messages = {
 };
 
 const TERMINATE_IDLE_PROCESS_TIMEOUT = 30 * 1000;
+const currentCLIMajorVersion = 0;
+// TODO: get this info from backend
+const currentCLIMinorVersion = 10;
```

Comment:
i think we can inject it with webpack define plugin instead of hardcoding https://webpack.js.org/plugins/define-plugin/

Reply:
i have a better idea. i will ping npm to fetch the latest version.